Source/ACE/Entity/WorldObject.cs

```diff
@@ -1963,7 +1991,7 @@ private WeenieHeaderFlag SetWeenieHeaderFlag()
             if (AmmoType != null)
                 weenieHeaderFlag |= WeenieHeaderFlag.AmmoType;
 
-            if (Value != null)
+            if (Value != null && (Value > 0))
```

Comment:
As long as we never need to send a value of 0 with the weenieheaderflag, then this is fine.